tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot

```diff
@@ -62,4 +65,18 @@ Wait Until Uninstallation Completes
 
 Operator Should Be Uninstalled
   [Arguments]  ${operator}
-  Page Should Not Contain Element  //a[@data-test-operator-row="${operator}"]
\ No newline at end of file
+  Page Should Not Contain Element  //a[@data-test-operator-row="${operator}"]
+
+Get RHODS version
+    #@{list} =  OpenShiftCLI.Get  kind=ClusterServiceVersion  label_selector=olm.copiedFrom=redhat-ods-operator
+    #&{dict} =  Set Variable  ${list}[0]
+    #Log  ${dict.spec.version}
+    ${ver} =  Run  oc get csv -n redhat-ods-operator | grep "rhods-operator" | cut -d' ' -f11
```

Comment:
This command returns an empty line in ods-qe-test2. Maybe you want to replace it for this one:
`oc get csv -n redhat-ods-operator | grep "rhods-operator" | awk '{print $1}' | sed 's/rhods-operator.//'`

Have you tried getting the version in PSI clusters? Does it work well there too?

Reply:
The command works on PSI for sure, I'll have to investigate why it's not working in that osd cluster

Reply:
For some reason the formatting of the output of 
`oc get csv -n redhat-ods-operator | grep "rhods-operator"`
is formatted differently between PSI and OSD. 

![Screenshot from 2021-11-26 11-41-49](https://user-images.githubusercontent.com/82518733/143568541-a296cdea-1061-42cc-a5f4-2dabee0e10d6.png)
![Screenshot from 2021-11-26 11-41-23](https://user-images.githubusercontent.com/82518733/143568542-ec8af5f8-9289-4699-afc2-69b2d9550149.png)

your variation of the command works on both, so I'll update to that

Reply:
Updated in latest commit

Reply:
Thanks!